### ISSUE TITLE
Certutils crash

### ISSUE THREAD

**Issue by Fuzion24** [mitmproxy/mitmproxy#1083]
```
Traceback (most recent call last):
  File "/usr/local/Cellar/mitmproxy/0.16/libexec/lib/python2.7/site-packages/libmproxy/proxy/server.py", line 121, in handle
    root_layer()
  File "/usr/local/Cellar/mitmproxy/0.16/libexec/lib/python2.7/site-packages/libmproxy/proxy/modes/http_proxy.py", line 11, in __call__
    layer()
  File "/usr/local/Cellar/mitmproxy/0.16/libexec/lib/python2.7/site-packages/libmproxy/protocol/tls.py", line 345, in __call__
    self._establish_tls_with_client_and_server()
  File "/usr/local/Cellar/mitmproxy/0.16/libexec/lib/python2.7/site-packages/libmproxy/protocol/tls.py", line 427, in _establish_tls_with_client_and_server
    six.reraise(*sys.exc_info())
  File "/usr/local/Cellar/mitmproxy/0.16/libexec/lib/python2.7/site-packages/libmproxy/protocol/tls.py", line 424, in _establish_tls_with_client_and_server
    self._establish_tls_with_client()
  File "/usr/local/Cellar/mitmproxy/0.16/libexec/lib/python2.7/site-packages/libmproxy/protocol/tls.py", line 433, in _establish_tls_with_client
    cert, key, chain_file = self._find_cert()
  File "/usr/local/Cellar/mitmproxy/0.16/libexec/lib/python2.7/site-packages/libmproxy/protocol/tls.py", line 566, in _find_cert
    return self.config.certstore.get_cert(host, list(sans))
  File "/usr/local/Cellar/mitmproxy/0.16/libexec/vendor/lib/python2.7/site-packages/netlib/certutils.py", line 341, in get_cert
    sans),
  File "/usr/local/Cellar/mitmproxy/0.16/libexec/vendor/lib/python2.7/site-packages/netlib/certutils.py", line 96, in dummy_cert
    ipaddress.ip_address(i.decode("ascii"))
AttributeError: 'NoneType' object has no attribute 'decode'

mitmproxy has crashed!
Please lodge a bug report at: https://github.com/mitmproxy/mitmproxy
```

**Comment by mhils** [comment]
This should be fixed in 0.17. Can you please update and try again?

**Comment by Fuzion24** [comment]
Looks fixed. Sorry, didn't see the updated version. Cheers.
